tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -1825,3 +1827,129 @@ def test_discard(self):
         assert tombstone.culprit == group1.culprit
         assert tombstone.project == group1.project
         assert tombstone.data == group1.data
+
+
+class GroupDeleteTest(APITestCase, SnubaTestCase):
```

Comment:
this also has a lot of duplication, but am planning on going back and refactoring later this week: https://getsentry.atlassian.net/browse/APP-1054